script.js

```diff
@@ -0,0 +1,194 @@
+"use strict";
+
+window.onload = async function () {
+  const filterCheckboxes = document.querySelectorAll(".input-default_checkbox");
+  const sortRadios = document.querySelectorAll(".input-default_radio");
```

Comment:
Имеет смысл обращаться к DOM-узлам при помощи селекторов, которые не связаны с его внешнем видом. Это нужно для того, чтобы изменение визуальной части шаблона не привело к поломке JS кода. Например, можно использовать специальные [data-* атрибуты](https://developer.mozilla.org/ru/docs/Web/Guide/HTML/Using_data_attributes)

```suggestion
  const sortRadios = document.querySelectorAll("[data-item='sort-option']");
```